src/public/js/rs.js

```diff
@@ -21,6 +105,94 @@ var RS = (function() {
     }, '');
   }
 
+  // Encodes using systematic BCH Reed-Solomon on Galois Field 256
```

Comment:
256 or 2^8? does the 2^8 notation mean that we're using an extension field over the base field GF(2) and the 256 notation mean that we're using an integer field? not sure whether that statement is true and not sure which way you're doing things (about to read the function)